src/fuel_fab.h

```diff
@@ -137,7 +137,7 @@ class FuelFab : public cyclus::Facility {
     "default": [], \
     "uilabel": "Filler Stream Preferences", \
     "doc": "Filler stream commodity request preferences for each of the given filler commodities (same order)." \
-           " If unspecified, default is to use 1.0 for all preferences.", \
+           " If unspecified, default is to use cyclus::kDefaultPref for all preferences.", \
```

Comment:
Since this is user doc, I'm not sure if I really like using the c++ dev constant.